ui/ui-frontend-common/src/app/modules/components/logbook-operation-facet/logbook-operation-facet.component.html

```diff
@@ -1,10 +1,14 @@
-<span class="title-text">{{ facetTitle }}</span>
-<div class="row row-data">
-  <div class="col title-text" *ngFor="let facet of facetDetails">
-    <div class="layout-data" [ngClass]="{ clickable: facet.clickable }" (click)="onFilter(facet)">
-      <span class="status-text" [ngStyle]="{ color: facet.color }">{{ facet.title }}</span>
-      <div class="result-text">
-        {{ facet.totalResults }}
+<div>
+  <span class="title-text" [ngStyle]="{'font-size': facetTitleSize + 'px'}">{{ facetTitle }}</span>
+  <div class="row row-data">
+    <div class="col title-text" *ngFor="let facet of facetDetails">
+      <div class="layout-data" [ngClass]="{ clickable: facet.clickable }" (click)="onFilter(facet)">
+        <div class="mb-2">
+          <span class="status-text" [ngStyle]="{ color: facet.color, 'font-size': facetSize + 'px' }">{{ facet.title }}</span>
+        </div>
+        <div class="result-text">
```

Comment:
Pourquoi avoir rendu les tailles paramétrables ? Dans quel contexte avons nous besoin de size différentes (pour rappel par soucis d'homogénéité de la charte graphique, nous ne sommes pas sensé override des styles mais plutôt utiliser des classes déjà prédéfinis).

Reply:
Il s'agit de l'existant, je vais faire un revert de cette partie faite en anticipation de la prochaine US.